onadata/libs/utils/csv_import.py

```diff
@@ -241,8 +241,25 @@ def submit_csv(username, xform, csv_file, overwrite=False):
     ona_uuid = {'formhub': {'uuid': xform.uuid}}
     error = None
     additions = duplicates = inserts = 0
+    xl_date_columns = []
+
     try:
-        for row in csv_reader:
+        for indx, row in enumerate(csv_reader):
+            #  check for any dates in first row
+            if indx == 0:
+                for key in list(row):
+                    if not key.startswith('_'):
+                        try:
+                            date = datetime.strptime(
+                                row.get(key, ''), '%d/%m/%Y')
```

Comment:
Is it possible to gate which fields are date fields from the `XForm` object?
How do we handle `datetime` fields?

Reply:
The `datetime` fields remain unchanged with `-` as it should in excel version 16.0 that I have tested with. So far, only `date` fields are affected. 
And it's possible to get the date fields from the `XForm`, and I think it could be more efficient so.